charclass_invlists.h

```diff
@@ -434684,7 +434684,7 @@ static const char * const UNI_lb_values[] = {
 	"infixnumeric",
 	"Inseparable",
 	"inseparable",
-	"Inseperable",
+	"Inseparable",
```

Comment:
a) this file is generated
b) the mis-spelling here is specified by [Unicode](http://www.unicode.org/Public/UCD/latest/ucd/PropertyValueAliases.txt).  Note the correct spelling is also available.

Reply:
change reverted, thanks